corehq/apps/domain/auth.py

```diff
@@ -300,11 +300,5 @@ def get_identifier(self, request):
 
         """
         username = self.extract_credentials(request)[0]
-        if API_THROTTLE_WHITELIST.enabled(username):
-            return username
-
-        try:
-            api_key = self.extract_credentials(request)[1]
-        except ValueError:
-            api_key = ''
-        return f"{getattr(request, 'domain', '')}_{api_key}"
+        domain = getattr(request, 'domain', '')
+        return ApiIdentifier(username=username, domain=domain)
```

Comment:
huh, I commented this a bit ago, but it disappeared - Stickler seems to be right about this one; ApiIdentifier is undefined.  That raises a question I had as well, is the non-tastypie implementation of this stuff tested?  The travis build passed.  I remember the PR where you implemented that stuff, but I don't recall whether it included tests.